packages/react-native-renderer/src/ReactNativeBridgeEventPlugin.js

```diff
@@ -31,6 +31,9 @@ const ReactNativeBridgeEventPlugin = {
     nativeEvent: AnyNativeEvent,
     nativeEventTarget: Object,
   ): ?Object {
+    if (targetInst == null) {
```

Comment:
This could use an inline comment about the case it's intended to handle. I'm not really familiar with this particular code, but it would not be obvious to me at a glance why it was okay to silently ignore this case.

Reply:
👍